### ISSUE TITLE
How to write `scientificName` for taxa with `taxonRank` = `speciesAggregate`

### ISSUE THREAD

**Issue by damianooldoni** [gbif/portal-feedback#4014]
We need to map some species aggregated taxa (see https://github.com/riparias/vmm-macrophytes-occurrences/issues/3#issuecomment-1092759261).

`speciesAggregate` seems a valid DwC term for `taxonRank` (see https://rs.gbif.org/vocabulary/gbif/rank_2015-04-24.xml). I just wonder how to write the scientificName of such aggregated species. In the link I provided, I read:
> Zoology: 'Aggregate - a group of species, other than a subgenus, within a genus. An aggregate may be denoted by a group name interpolated in parentheses.'

Something like `Callitriche (obtusangula,platycarpa)`? Maybe it would be nice to add some suggestions/examples in documentation.

Thanks a lot.

**Comment by peterdesmet** [comment]
Another common use case are observations where the observer hesitates between two or more species:

> Branta canadensis / Branta hutchinsii

One could provide the genus (`Branta`), but then one looses originally provided information. What is the best way to write the scientific name here? And what taxonRank should be assigned?

**Comment by AbbyBenson** [comment]
I'm not sure if this is correct but in cases like Branta canadensis/hutchinsii I put "Branta" in `scientificName` and "canadensis/hutchinsii" in `identificationQualifier` and `taxonRank` = "genus".

**Comment by ManonGros** [comment]
@mdoering do you have any recommendation on how to write scientific names for species aggregate?

**Comment by mdoering** [comment]
For species aggregates / species groups we recommend `agg.` as the rank marker and treat it otherwise just like a binomial.
The [name parser rank enumeration](https://github.com/gbif/name-parser/blob/master/name-parser-api/src/main/java/org/gbif/nameparser/api/Rank.java#L132) contains this:

>   /**
>    * A loosely defined group of species, often in flux.
>    * Often also called species complex, or superspecies.
>    */
>   SPECIES_AGGREGATE("agg."),

and contains some [tests for aggregate names](https://github.com/gbif/name-parser/blob/master/name-parser-api/src/test/java/org/gbif/nameparser/NameParserTest.java#L846) that can be parsed:

 - Achillea millefolium agg. L.
 - Strumigenys koningsbergeri-group
 - Selenophorus parumpunctatus species group
 - Monomorium monomorium group

`group` is another often found marker that the parser understands.

The species aggregate should use a single epithet/word for the aggregate name, otherwise it will not be parsable and much harder to compare. `Hypsiboas calcaratus species complex` is being used in `[this paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3904076/)` which should be parsable as it uses `calcaratus` as the name for the complex.

There is no support or even recommendation on how to deal with identifications to multiple taxa. This is a long outstanding issue which needs both  guidelines and support/implementation in a whole range of GBIF software. It is not an easy feature to add, but I reckon well worth a coordinated effort.

**Comment by peterdesmet** [comment]
Thanks @mdoering that clarifies how to submit species groups. I notice they get exact matched to the species:

`Ranunculus peltatus group` -> [Ranunculus peltatus Schrank](https://www.gbif.org/species/8365713)

Why I assume is intended behaviour.

> This is a long outstanding issue which needs both guidelines and support/implementation in a whole range of GBIF software. It is not an easy feature to add, but I reckon well worth a coordinated effort.

Would it be useful to create a separate issue for those, so this one can be closed?

**Comment by mdoering** [comment]
Yes, lets open a new one for IDs to multiple taxa. The exact match of the species group to the regular species might not be ideal. We don't include any aggregates in the backbone, but I think a fuzzy match would be better in that case!

**Comment by peterdesmet** [comment]
Issue about 2 provides names now recorded at https://github.com/gbif/portal-feedback/issues/4023. The remaining task in this issue is to treat names that contain `group` as a fuzzy rather than exact match.
